ui/puzzle/src/ctrl.ts

```diff
@@ -50,6 +50,7 @@ export default function (opts: PuzzleOpts, redraw: Redraw): Controller {
   // required by ceval
   vm.showComputer = () => vm.mode === 'view';
   vm.showAutoShapes = () => true;
+  vm.delayedStart = opts.pref.animation.duration > 0;
```

Comment:
this new property looks superfluous, could read `opts` directly from the `setTimeout` instead?